packages/react/src/components/Notification/Notification.js

```diff
@@ -396,6 +419,8 @@ ToastNotification.defaultProps = {
 };
 
 export function InlineNotification({
+  ['aria-label']: ariaLabel,
+  ariaLabel: deprecatedAriaLabel,
```

Comment:
I'm thinking since we are adding this prop to `InlineNotification` for the first time now, we can probably get away with not adding `ariaLabel` at all and just adding `aria-label` here. I don't think it's necessary to add a deprecated prop here that wasn't supported previous to this PR 🤔

Reply:
Yeah that makes sense. The only reason I could think to have it in there is so that the notification types are uniform. Would you like for me to remove `ariaLabel`?

Reply:
Yeah I say let's go ahead and remove it. thanks! @jsehull